docs/DeveloperGuide.md

```diff
@@ -283,32 +283,422 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `Healthcare Xpress` and the **Actor** is the `medical administrator`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC01 - Delete a patient/nurse**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  Medical administrator requests to list patients,nurses or both according to general or specific groups.
+2.  Healthcare Xpress shows a list of patients/nurses that satisfy the group's condition.
+3.  Medical administrator requests to delete a specific patient/nurse in the list.
+4.  Healthcare Xpress deletes the patient/nurse.
 
     Use case ends.
 
 **Extensions**
+* 1a. The given input is invalid.
 
+    * 1a1. Healthcare Xpress shows an error message.
+
+      Use case restart. 
+
+* 1b. Medical administrator get a specific patient/nurse.
+
+    * 1b1. Healthcare Xpress shows that patient/nurse.
+  
+    * 1b2. Medical administrator requests to delete that patient/nurse.
+  
+      Use case resumes from step 4.
+    
 * 2a. The list is empty.
 
   Use case ends.
 
 * 3a. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. Healthcare Xpress shows an error message.
 
       Use case resumes at step 2.
 
-*{More to be added}*
+* a. At any time, Medical administrator choose to exit the program.
+  
+  Use case ends. 
+
+
+**Use case: UC02 - Mark a patient**
+
+**MSS**
+
+1.  Medical administrator requests to list patients according to general or specific groups.
+2.  HealthCare Xpress shows a list of patients that satisfy the group's condition.
+3.  Medical administrator requests to mark a specific patient in the list.
+4.  HealthCare Xpress mark the patient as visited.
+
+    Use case ends.
+
+**Extensions**
+* 1a. The given input is invalid.
+
+    * 1a1. Healthcare Xpress shows an error message.
+
+      Use case restarts.
+
+* 1b. Medical Administrator request to list nurses only.
+
+    Use case ends. 
+
+* 1c. Medical Administrator request to get a specific patient.
+
+    * 1c1. Healthcare Xpress shows that patient. 
+
+    * 1c2. Medical administrator requests to mark that patient.
+
+      Use case resumes from step 4.   
```

Comment:
As mentioned above I'm not sure if this can be omitted. Let me know what you think!